components/finance/FinanceCategoriesListItem/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const FinanceCategoriesListItem = ({ category }: any) => (
+  <>
+    {category &&
+      category.map((item: any) => (
+        <li key={item.name} className="flex mt-px">
```

Comment:
do not add li inside a component so that the component can be used independently of a <ul> tag

Reply:
fixed. 
I assumed that It will always be a list that will be rendered through this component.